index.html

```diff
@@ -87,7 +87,7 @@ <h2>Project name goes here</h2>
       <div class="container-2">
         <div class="group-2-3">
           <div class="project-container project-2">
-            <img class="project" src="images/project-2.jpg" alt="project-2 image">
+            <img class="project" src="images/project-2.jpg" alt="macbook pro 16 frontal view">
 
             <div class="single-project-info">
                   <h2>Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.